src/AspNet.Security.OAuth.VkId/VkIdAuthenticationHandler.cs

```diff
@@ -19,12 +19,16 @@
 
 namespace AspNet.Security.OAuth.VkId;
 
-public sealed class VkIdAuthenticationHandler(
-    IOptionsMonitor<VkIdAuthenticationOptions> options,
-    ILoggerFactory logger,
-    UrlEncoder encoder)
-    : OAuthHandler<VkIdAuthenticationOptions>(options, logger, encoder)
+public sealed class VkIdAuthenticationHandler : OAuthHandler<VkIdAuthenticationOptions>
 {
+    public VkIdAuthenticationHandler(
+        IOptionsMonitor<VkIdAuthenticationOptions> options,
+        ILoggerFactory logger,
+        UrlEncoder encoder)
```

Comment:
To match the other implementations.

```suggestion
        IOptionsMonitor<VkIdAuthenticationOptions> options,
        ILoggerFactory logger,
        UrlEncoder encoder)
```
```suggestion
        [NotNull] IOptionsMonitor<VkIdAuthenticationOptions> options,
        [NotNull] ILoggerFactory logger,
        [NotNull] UrlEncoder encoder)
```